FreeRTOS/FreeRTOS-Kernel
# Style: Make freertos.org = FreeRTOS.org and add https

Style: make freertos.org = FreeRTOS.org and add https

Description
-----------
Make freertos.org = FreeRTOS.org and add https in comments
```grep 'http://www.freertos.org' . -RIl | xargs sed -i -e 's#http://www.freertos.org#https://www.FreeRTOS.org#g'
```
Test Steps
-----------
no source code has been modified

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
